Project Code/views/pages/login.ejs

```diff
@@ -1,19 +1,30 @@
 <%- include ('../partials/head') %>
 <%- include ('../partials/nav') %>
 
-<main>      
-    <div class="border"style="text-align: center; margin-left:auto; margin-right:auto;width:500px; margin-top:150px">
-     <h1 style="margin-top:30px">Login</h1>
+<main>
+    <style>
+        label {
+            display: inline-block;
+            width: 80px;
+            text-align: right;
+        }
+    </style>
+    <div class="border"
+        style="text-align: center; margin-left:auto; margin-right:auto;width:500px; margin-top:150px">
+        <h1 style="margin-top:30px">Login</h1>
         <form action="/login" method="POST" style="margin-top:50px">
             <div style="margin-bottom:20px">
-            <label for="name" >Username: </label>
-            <input required type="text"  id="username" name="username" />
-        </div>
-        <div>
-            <label for="password">Password:  </label>
-            <input required type="text" id="password" name="password" />
-        </div>
-            <input style="margin-top:60px; margin-bottom: 20px;"type="submit" value="Login" />
+                <label for="name">Username: </label>
+                <input required type="text" id="username" name="username" />

```

Comment:
Should we go for email instead of username?

Reply:
Yeah email sounds good I'll change it right quick